acc_test/data/valid_multi_dc_provisioning.tf

```diff
@@ -0,0 +1,68 @@
+provider "instaclustr" {
```

Comment:
[question] maybe we should have an invalid test covering multi dc?

Reply:
sure thing, will add the invalid ones.